src/reporter/tests/geo_queries_fixture.py

```diff
@@ -42,6 +43,7 @@ def setup_entities():
 
 
 def run_query(service, base_url, query_params, expected_status_code=200):
+    time.sleep(1)
```

Comment:
ouch! you sure we need this? if sync among services is what we need, we should try avoiding `sleep` as much as we can---see #447. It's just too unreliable---in general you can't solve coordination problems w/ time waits. It took us a loooong time to fix timing issues in reporter tests---see #574. So going forward, I'd rather write code that takes into account test execution, data load and asserts---see e.g. the wait functions in `reporter.tests.utils`.